tools/stringtable_validator.py

```diff
@@ -162,13 +162,11 @@ def main():
     bad_count = 0
 
     for filepath in stringtable_files:
-        print("\nChecking {}:".format(os.path.relpath(filepath, root_dir)))
+        print("Checking {}:".format(os.path.relpath(filepath, root_dir)))
 
         errors = check_stringtable(filepath)
 
-        if errors == 0:
-            print("No errors found.")
-        else:
+        if errors != 0:
```

Comment:
I'd prefer to keep the more nicely formatted output. We don't have too many components so it works fine.